### PR TITLE
Form controls

### PR DESCRIPTION
**Pull request by NejcZdovc**
- Move control wrapper into separate component
- Removes customStyle
- Sync with design
- Adds OFF to toggle component
- Removes asset folders (replaces with icons components)
- Add floating style to select

**This PR is not using predefined colors yet**

### CODE REVIEW

**Comment by NejcZdovc** [comment]
There is one TODO that is pending icon update, but everything can be reviewed.

**Comment by petemill** [comment]
@NejcZdovc updated icons for carat and arrow generated from latest sketch (to contain less built-in padding), using the tool in #91.

I rebased your commit for this PR on top of my new icon one, in case you need to amend yours with further changes.

Seems like the storybook build is failing for this branch: https://brave-ui-kyysxvvdsh.now.sh

**Comment by NejcZdovc** [comment]
@petemill it's failing because of the new icons, will update spec

File: src/components/formControls/toggle/style.ts [hunk 1]
```diff
@@ -4,127 +4,83 @@
...
-    ` : ''
-    };
-
-  ${(p: Props) => p.checked && p.size === 'large'
-    ? css`
-      transform: translate(16px, calc(-50% - 4px));
-    ` : ''
-    };
-` as any
+  ${getBulletStyle};
```
Comment (petemill) [inline src/components/formControls/toggle/style.ts]: same re: disjointed css rules

File: src/components/formControls/toggle/style.ts [hunk 2]
```diff
@@ -4,127 +4,83 @@
...
+`
+
+export const StyledText = styled<Props, 'div'>('div')`
+  color: #838391;
+  font-size: ${(p: Props) => p.size === 'small' ? '12px' : '14px'};
+  font-family: Poppins, sans-serif;
+  text-align: right;
+  letter-spacing: 0.4px;
+  text-transform: uppercase;
+  margin: ${(p: Props) => p.size === 'small' ? '0px' : '4px'} 8px 0 0;
```
Comment (petemill) [inline src/components/formControls/toggle/style.ts]: Don't need to re-define type for every child function

File: src/components/formControls/select/style.ts [hunk 3]
```diff
@@ -3,52 +3,82 @@
...
-  background: #fff;
   width: 100%;
-  padding: ${(p: {show: boolean, customStyle: Theme}) => setTheme(p.customStyle, 'padding') || '10px 0'};
-  border: ${(p: {show: boolean, customStyle: Theme}) => setTheme(p.customStyle, 'border') || '1px solid #dfdfe8'};
+  padding: ${(p: Props) => p.floating ? 0 : 8}px 0;
   display: flex;
   align-items: center;
+  ${getSelectColors};
+`
+// TODO update width when we get new icons
```
[the remark this answers is not in the document]
Comment (NejcZdovc) [reply src/components/formControls/select/style.ts]: yup, forgot to remove this comment

File: src/components/formControls/checkbox/style.ts [hunk 4]
```diff
@@ -3,42 +3,81 @@
...
+export const StyledBox = styled<Partial<StyleProps>, 'span'>('span')`
   border-radius: 2px;
-  border: solid 1px ${(p: {selected: boolean, customStyle: Theme}) => setTheme(p.customStyle, 'borderColor') || '#d1d1db'};
-  margin-right: 11px;
   text-align: center;
   display: flex;
   align-items: center;
   justify-content: center;
+  padding: 0 2.5px;
+  ${getBox};
```
[the remark this answers is not in the document]
Comment (NejcZdovc) [reply src/components/formControls/checkbox/style.ts]: done

File: src/components/formControls/select/style.ts [hunk 5]
```diff
@@ -3,52 +3,82 @@
...
+      borderColor = '#686978'
+    }
+  }
+
+  let border = `1px solid ${borderColor}`
+  if (p.floating) {
+    border = 'none'
+  }
+
+  return css`
```
[the remark this answers is not in the document]
Comment (NejcZdovc) [reply src/components/formControls/select/style.ts]: done

File: src/components/formControls/toggle/style.ts (hunk 1 above)
Comment (NejcZdovc) [reply src/components/formControls/toggle/style.ts]: done

**Pull request merged**